modules/flowable-ui/flowable-ui-modeler-frontend/src/main/resources/static/modeler/i18n/en.json

```diff
@@ -730,6 +730,20 @@
             "DESCRIPTION" : "Define the error name."
           }
         },
+        "ERRORVARIABLEPACKAGE" : {
+          "ERRORVARIABLENAME" : {
+            "TITLE" : "Error variable name",
+            "DESCRIPTION" : "Define the error variable name."
+          },          
+          "ERRORVARIABLETRANSIENT" : {
+            "TITLE" : "Store as transient variable",
+            "DESCRIPTION" : "Flag defines weather a variable will be stored transiently."
+          },
+          "ERRORVARIABLELOCALSCOPE" : {
+            "TITLE" : "Store to a local scope",
+            "DESCRIPTION" : "Flag defines weather variable will be stored to a local scope."
```

Comment:
Spelling mistake: weather -> whether

Reply:
Ups! Embarrassing ... will fix it at once :)